### DISCUSSION TITLE
Release 73

### DISCUSSION THREAD

**Discussion by oliverw** [coinfoundry/miningcore#1426] [Announcements]
- Disconnect DAG generation from miner connection state. Fixes #1395
- Fix for Ethereum stratum difficulty changes not sent to miners. Fixes #1404
- Fix Ergo block effort calculation. Fixes #1405
- Add support for ETHW (Ethereum POW fork) 
- Update for Monero 18. Fixes #1175 
- Update Linux build scripts and add script for Ubuntu 22.04
- Reduce memory allocation in stratum collection to reduce GC pressure

<hr /><em>This discussion was created from the release <a href='https://github.com/oliverw/miningcore/releases/tag/v73'>Release 73</a>.</em>

**Comment by feculay** [comment]
hi, how to config config.json for ETHW (Ethereum POW fork)?, thanks

**Comment by n1ckl4s** [comment]
Excuse the newbie question but how do I do the MC update? so you can get the upgrades.

**Reply by LoulouCrypto** [reply]
With ``git pull `` and you have to recompile it

**Reply by n1ckl4s** [reply]
Thanks for replying, so with that I have to reinstall everything I guess.
